tests/Unit/Lock/unlock_host_pointer.cpp

```diff
@@ -0,0 +1,34 @@
+// XFAIL: Linux
+// RUN: %hc %s -o %t.out %t.out
```

Comment:
@adityaatluri same here. perhaps -lhc_am is needed so lit could correct build the test